src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -154,8 +154,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
         c.argument('account_type', help='The storage account type', arg_type=get_enum_type(t_account_type))
         c.argument('account_name', acct_name_type, options_list=['--name', '-n'], completer=None)
-        c.argument('kind', help='Indicates the type of storage account.',
-                   arg_type=get_enum_type(t_kind, default='storage'))
+        c.argument('kind', help='Indicates the type of storage account.', min_api="2017-10-01",
+                   arg_type=get_enum_type(t_kind), default='StorageV2')
+        c.argument('kind', help='Indicates the type of storage account.', max_api="2017-06-01",
+                   arg_type=get_enum_type(t_kind), default='Storage')
```

Comment:
>Storage [](start = 60, length = 7)

the default value was "storage" while it is "Storage" now, is it by design?

Reply:
In cli commands, if you use get_enum_types(), the input value will be case insensitive. In this way, it is doesn't matter for "storage: or "Storage" here.

Reply:
so this default value is only used by our code, it is not visible to user?

---
In reply to: [376912505](https://github.com/Azure/azure-cli/pull/12112#discussion_r376912505) [](ancestors = 376912505)